test/studies/shootout/meteor/kbrady/meteor-parallel-alt.chpl

```diff
@@ -335,13 +345,27 @@ proc searchLinearHelper(in board, in pos, in used, in placed,
 }
 
 
+// DIY sync variable functionality that outperforms native sync variables.
+// Access controlled by functions lock() and unlock()
+var l: atomic bool;
+
+
+
+
```

Comment:
Lots of extra lines here.  There's two before this declaration, but four after it